README.md

```diff
@@ -1,132 +1,210 @@
-A Procedural Language in JavaScript powered by V8
-=================================================
-
-plv8 is a shared library that provides a PostgreSQL procedural language powered
-by V8 JavaScript Engine.  With this program you can write in JavaScript your
-function that is callable from SQL.
-
-REQUIREMENT
------------
-
-plv8 is tested with:
-
+A Procedural Language in JavaScript powered by V8
+=================================================
+
+plv8 is a shared library that provides a PostgreSQL procedural language powered
+by V8 JavaScript Engine.  With this program you can write in JavaScript your
+function that is callable from SQL.
+
+# Requirements:
+plv8 is tested with:
```

Comment:
@JerrySievert Do you know if the:
```
REQUIREMENT
-----------
```

Format is needed for the PGXN wiki or if I can use the Markdown `#` formatter for titles?
```
# Requirements:
```